### PR TITLE
Improve node.js devfile

### PR DESCRIPTION
**Pull request by AndrienkoAleksandr**
### What does this PR do?
Enhancement: Add stop nodejs server command.

Signed-off-by: Oleksandr Andriienko <oandriie@redhat.com>

### CODE REVIEW

File: devfiles/nodejs/devfile.yaml [hunk 1]
```diff
@@ -23,10 +23,22 @@ components:
...
     actions:
       - type: exec
         component: nodejs
-        command: nodemon app.js
-        workdir: ${CHE_PROJECTS_ROOT}/nodejs-web-app/app
+        command: >-
+          if [ ! -d 'node_modules' ]; then echo 'Install node dependencies...';
+          npm install; fi; nodemon app/app.js
+        workdir: '${CHE_PROJECTS_ROOT}/nodejs-web-app'
+
```
Comment (sleshchenko) [inline devfiles/nodejs/devfile.yaml]: Do you think we need this empty line here?

File: devfiles/nodejs/devfile.yaml [hunk 2]
```diff
@@ -23,10 +23,22 @@ components:
...
-  -
+  - 
     name: run the web app
     actions:
       - type: exec
         component: nodejs
-        command: nodemon app.js
-        workdir: ${CHE_PROJECTS_ROOT}/nodejs-web-app/app
+        command: >-
+          if [ ! -d 'node_modules' ]; then echo 'Install node dependencies...';
```
Comment (sleshchenko) [inline devfiles/nodejs/devfile.yaml]: Would not it better to format it in the following way?
```bash
  if [ ! -d 'node_modules' ]; then 
    echo 'Install node dependencies...';
    npm install; 
  fi; 
  nodemon app/app.js
```
Or
```bash
  if [ ! -d 'node_modules' ]; then echo 'Install node dependencies...'; npm install; fi; 
  nodemon app/app.js
```

**Comment by ibuziuk** [comment]
@AndrienkoAleksandr could you please update the PR since devfile has been updated with debug config and 'install dep' command. Stop command in devfile absolutely makes sense until https://github.com/eclipse/che/issues/13737#issuecomment-493868402 is fixed on IDE side

**Comment by AndrienkoAleksandr** [comment]
@ibuziuk @sleshchenko done.

**Comment by ibuziuk** [comment]
@skabashnyuk @rhopp @vparfonov could you please provide codeowners' approval ?

**Pull request merged**